run_viralrecon.sh

```diff
@@ -36,7 +36,7 @@ echo "$(date) : First create input samplesheet for viralrecon pipeline"
 if [ ! -f "$out_dir/${run_name}_samplesheet.csv" ]
     then
         echo "$(date) : $infile does not exist. Creating samplesheet required to run viralrecon"
-        python $script_dir/conf-files/fastq_dir_to_samplesheet.py $ww_fastq $out_dir/$infile
+        /home/linuxbrew/.linuxbrew/bin/python3 $script_dir/conf-files/fastq_dir_to_samplesheet.py $ww_fastq $out_dir/$infile
```

Comment:
Does this need a specific version of python or a specific library?

I ask, because we're updating the operating system in may and this path may change.

Reply:
I needed to specify a specific python version to make sure it works for other users. I had to modify this when Olinto and Tom ran this script when I was away.
I think the issue was due to multiple python versions on the LW.

I can revisit this when we update the OS.